### PR TITLE
STEM: Update form 22-10203 submission. #7857

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/7857
Schema changes https://github.com/department-of-veterans-affairs/vets-json-schema/pull/448

## Screenshots
image
[Veteran Request For Change _ Veterans Affairs.pdf](https://github.com/department-of-veterans-affairs/vets-website/files/4886234/Veteran.Request.For.Change._.Veterans.Affairs.pdf)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -258,7 +258,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "6.9.2",
+    "@department-of-veterans-affairs/formation": "6.9.3",
```
Comment (U-DON) [inline package.json:261]: I would prefer to see dependency upgrades as separate PRs.

Comment (zurbergram) [reply package.json:261]: https://github.com/department-of-veterans-affairs/vets-website/pull/13405

Comment (zurbergram) [reply package.json]: pulled in #13405 to this branch

**Review by Mottie — APPROVED** [review]
It looks like we don't need to include

```css
.schemaform-title,
.schemaform-subtitle {
  display: none;
}
```

in the css either - see https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/modules/_m-print.scss#L12-L16

**Review by U-DON — APPROVED** [review]

**Pull request merged**